tests/test_api.rs

```diff
@@ -415,6 +415,50 @@ fn file_post_handles_errors() {
     validate_error(&body, Some("invalid hash 'badhash'"));
 }
 
+#[test]
+fn can_post_metadata() {
+    let root = get_test_dir();
+    let rocket = outpack::api::api(root.clone());
+    let client = Client::tracked(rocket).expect("valid rocket instance");
+    let content = "{
```

Comment:
https://rahul-thakoor.github.io/rust-raw-string-literals/ this might be nicer for the embedded quotes